src/main.cpp

```diff
@@ -4456,23 +4454,8 @@ void GridcoinServices()
         }
     }
 
-    if (KeyEnabled("exportmagnitude"))
-    {
-        if (TimerMain("export_magnitude",900))
-        {
-            json_spirit::Array results;
-            results = MagnitudeReportCSV(true);
-
-        }
-    }
-
     if (TimerMain("gather_cpids",480))
-    {
-            //if (fDebug10) printf("\r\nReharvesting cpids in background thread...\r\n");
-            //LoadCPIDsInBackground();
-            //printf(" {CPIDs Re-Loaded} ");
-            msNeuralResponse="";
-    }
+        msNeuralResponse.clear();
```

Comment:
why you removed the barackets?

Reply:
Because one-line brackets feel too fluffy :)